modules/core/src/main/java/org/apache/synapse/message/StoreForwardException.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.synapse.message;
+
+/**
+ * Created by admin on 1/12/18.
```

Comment:
Shall we add a proper class level comment?

Reply:
Will add a class level comment.